src/main/java/seedu/address/logic/commands/RemoveCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.deadline.Deadline;
+
+//@@author:dalsontws
+/**
+ * Deletes a flashCard identified using it's displayed index from the address book.
+ */
+public class RemoveCommand extends Command {
```

Comment:
Personally, I feel that the naming is a bit ambiguous. Maybe it can change to deleteDeadline or something as remove could also mean remove a flashcard?